README.rst

```diff
@@ -37,35 +37,45 @@ Some noteworthy flags::
 
     codespell -w, --write-changes
 
-The -w flag will actually implement the changes recommended by codespell. Not running with ``-w`` flag is the same as with doing a dry run. It is recommended to run this with the ``-i`` or ``--interactive`` flag.::
+The ``-w`` flag will actually implement the changes recommended by codespell. Not running with ``-w`` flag is the same as with doing a dry run. It is recommended to run this with the ``-i`` or ``--interactive`` flag.::
 
     codespell -I FILE, --ignore-words=FILE
 
-The -I flag can be used to whitelist certain words that are in the ``codespell_lib/data/dictionary.txt``. The format of the whitelist file is one word per line. Invoke using: ``codespell -I path/to/file.txt`` to execute codespell referencing said whitelist. **Important note:** The whitelist passed to ``-I`` is case-sensitive based on how it is listed in ``dictionary.txt``. ::
+The ``-I`` flag can be used to whitelist certain words that are in the ``codespell_lib/data/dictionary.txt``. The format of the whitelist file is one word per line. Invoke using: ``codespell -I path/to/file.txt`` to execute codespell referencing said whitelist. **Important note:** The whitelist passed to ``-I`` is case-sensitive based on how it is listed in ``dictionary.txt``. ::
 
     codespell -L word1,word2,word3,word4
 
-The -L flag can be used to whitelist certain words that are comma-separated placed immediately after it. ::
+The ``-L`` flag can be used to whitelist certain words that are comma-separated placed immediately after it. ::
 
     codespell -S, --skip=
 
-Comma-separated list of files to skip. It accepts globs as well.  Examples:
+The ``-S`` (``--skip``) flag is used to skip over a comma-separated list of files and/or directories. It accepts globs as well. **Important note:** Do not append a ``/`` (forward-slash) to a directory you want to skip, this renders the entry for that particular directory ineffective. ::
 
-* to skip .eps & .txt files, invoke ``codespell --skip="*.eps,*.txt"``
+The ``-i <number>`` flag is used to invoke 'interactive mode' when writing changes. 0: no interactivity. 1: ask for confirmation. 2: ask user to choose one fix when more than one is available. 3: will combine both options 1 and 2 together.
```

Comment:
Move this up with the bit that recommends interactive mode (or add a note there saying see below?